crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -187,6 +202,31 @@ impl Reader {
     }
 }
 
+#[instrument(skip_all, err)]
+async fn get_celestia_chain_id(celestia_client: &CelestiaClient) -> eyre::Result<String> {
```

Comment:
Just return the (celestia_tendermint specific) `chain::Id` similar to what `get_sequencer_chain_id` further down is doing.